rdk/template/configRuleOrganization.yaml

```diff
@@ -0,0 +1,293 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: AWS CloudFormation template to create custom AWS Config rules. You
+  will be billed for the AWS resources used if you create a stack from this
+  template.
+Parameters:
+  RuleName:
+    Description: Name of the Rule
+    Type: String
+    MinLength: "1"
+    MaxLength: "128"
+  Description:
+    Description: Description of the Rule
+    Type: String
+    MinLength: "1"
+    MaxLength: "255"
+  RuleLambdaName:
+    Description: Name of the Rule's Lambda function
+    Type: String
+    MinLength: "1"
+    MaxLength: "64"
+  LambdaRoleArn:
+    Description: ARN of the existing IAM role that you want to attach to the lambda
+      function.
+    Type: String
+    Default: ""
+  BoundaryPolicyArn:
+    Description: ARN of a Boundary Policy, will be used only if LambdaRoleArn is NOT set.
+    Type: String
+    Default: ""
+  SourceBucket:
+    Description: Name of the S3 bucket that you have stored the rule zip files in.
+    Type: String
+    MinLength: "1"
+    MaxLength: "255"
+  # SourcePath:
```

Comment:
Why was this commented out?

Reply:
Also commented out in 2 additional locations in the code base.

Reply:
It was not used in the CFT anywhere, so I think it's just bloated old code that we can eventually get rid of.